### DISCUSSION TITLE
Love it

### DISCUSSION THREAD

**Discussion by bsless** [borkdude/quickdoc#15] [Show and tell]
Wanted to say I just started using quickdoc and I already love it. Great stuff!
Thanks, Borkdude!

https://github.com/bsless/tools.jvm
https://github.com/bsless/tools.jcmd.jfr

**Comment by borkdude** [comment]
Thanks! I get a 404 on tools.jvm btw, maybe it's not public?

**Reply by bsless** [reply]
lol, thanks, fixed
